config/config.go

```diff
@@ -869,6 +870,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.lockerdome.endpoint", "https://lockerdome.com/ladbid/prebidserver/openrtb2")
 	v.SetDefault("adapters.logicad.endpoint", "https://pbs.ladsp.com/adrequest/prebidserver")
 	v.SetDefault("adapters.lunamedia.endpoint", "http://api.lunamedia.io/xp/get?pubid={{.PublisherID}}")
+	v.SetDefault("adapters.madvertise.endpoint", "https://mobile.mng-ads.com/bidrequest{{.ZoneID}}")
```

Comment:
We have a limited amount of predefined macros and `PlacementID` isn't among them. Not ideal, but also perhaps a larger ask to have each adapter add their specific macro terminology. Not sure how to do this in a more generic way.

Reply:
Because I use default value https://pkg.go.dev/github.com/prebid/prebid-server/macros#ResolveMacros and ZoneID looks like PlacementID

Instead of add PlacementID on EndpointTemplateParams I user ZoneID on my adaper. I push changes.

Now bidRequest is 

`curl --request POST \
 --url 'http://localhost:8000/openrtb2/auction' \
 --header 'Content-Type: application/json' \
 --header 'X-Openrtb-Version: 2.5' \
 --header 'X-Forwarded-For: 91.165.240.140' \
 --data '{
   "id": "some-request-id",
   "test": 1,
   "site": {
     "page": "prebid.org"
   },
   "imp": [{
     "id": "some-impression-id",
     "banner": {
       "format": [{
         "w": 600,
         "h": 500
       }, {
         "w": 300,
         "h": 600
       }]
     },
     "ext": {
       "prebid": {
         "bidder": {
           "madvertise": {
             "zoneId": "/1753588/banner"
           }
         }
       }
     }
   }],
   "tmax": 1000
 }'`

Reply:
[master ed2ec56a] rename PlacementID to ZoneID

Reply:
Ah! i forgot about thee EndpointTemplateParams. Yeah it would definitely be great to have the ability for each adapter to be able to add their specific macro terminology but for now maybe we can add a new parameter PlacementID in EndpointTemplateParams for now rather than using ZoneID. This is not ideal but renaming the parameter to ZoneID for the sake of consistency in code is good but for publishers this might be confusing as the are actually specifying placement ID.

Thoughts @SyntaxNode ?

Reply:
I replace PlacementID by ZoneID on my code and docs. not a problem for me. I have pushed changes

Reply:
Ah ok! That's not what we intended but as long as you're okay with it and it won't be too confusing for the publishers to supply a Placement ID in a Zone ID param, we're good

Reply:
I'd also like to ad that it wasn't our intent for you to rename the bidder parameters for your adapter from placementID to zoneID. I'd hate to cause confusion to your publishers just for limitations in the PBS macro replacement options. You are certainly welcomed to keep this as zoneID if you wish or revert to placementID. 

I'm uenresvoling this issue just to ensure you see this comment.

Reply:
not a problem for me to keep ZoneID